src/server/test/web/readingsLineGroupFlow.js

```diff
@@ -26,8 +26,101 @@ mocha.describe('readings API', () => {
     mocha.describe('readings test, test if data returned by API is as expected', () => {
         mocha.describe('for line charts', () => {
             mocha.describe('for flow groups', () => {
-
                 // Add LG8 here
+                mocha.it(
+                    "LG8: should have daily points for 15 + 20 minute reading intervals and flow units with +-inf start/end time & kW as kW",
+                    async () => {
+                        const unitDatakWh = [
```

Comment:
I think it would be good to change unit, conversion, meter and group so the name reflects the actual unit. I also think this is a good idea so standard names already defined by OED are not change. Thus, kWh becomes kW in all the identifiers, e.g., unitDatakWh -> unitDatakW.